lib/logger.js

```diff
@@ -0,0 +1,55 @@
+const debugLib = require('debug')
+const errorLogger = debugLib('electronpdf:error:')
+const infoLogger = debugLib('electronpdf:info:')
+const debugLogger = debugLib('electronpdf:debug:')
+
+const LEVELS = {
```

Comment:
I could do that for the external argument passed in.  But for the internal representation my intent was to align to the npm log levels https://github.com/winstonjs/winston#logging-levels which I noticed I have wrong, so I will fix that.